Package.swift

```diff
@@ -38,236 +38,10 @@ let package = Package(
 // ** ⚠️ GENERATED, do not edit directly below this point **
 
 var streamChatSourcesExcluded: [String] { [
-    "Database/DataStore_Tests.swift",
-    "Database/DatabaseSession_Mock.swift",
-    "Database/DatabaseSession_Tests.swift",
-    "Database/DTOs/UserDTO_Tests.swift",
-    "Database/DTOs/ChannelMuteDTO_Tests.swift",
-    "Database/DTOs/ChannelDTO_Tests.swift",
-    "Database/DTOs/CurrentUserDTO_Tests.swift",
```

Comment:
Don't we need to update the script that does this?

Reply:
Done. script removed!

Reply:
I think the script still needs to exist, but we need now to exclude the snapshots in the test target 🤔

Reply:
The only reason the script was in place was to exclude the tests from the Swift Package.

Fortunately, with this change this is no longer needed. Also our Package manifest became more readable :)

Reply:
Yes I just saw, that, nevermind, there are no Test targets in the Package.swift 👍

Reply:
I did find something tho 🤔 I was testing the SPM Integration project, and it seems the local repo doesn't get updated at all. So the SPM Integration job is actually not doing anything 😞 But ofc, this out of scope of the PR, but is something we need to keep in mind

Reply:
I can look on that part in one of the following PRs ;)